app/models/user.rb

```diff
@@ -38,4 +40,8 @@ class User < ApplicationRecord
   # :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable, :confirmable,
          :recoverable, :rememberable, :validatable
+
+  def company?
```

Comment:
please, don't right such methods in model. Decorator is better place, check SimpleDelegator